### ISSUE TITLE
docker-compose up fails

### ISSUE THREAD

**Issue by jmandel** [siteadmin/inferno#71]
```
$ sudo docker-compose  up
Starting inferno_ruby_server_1 ... 
Starting inferno_ruby_server_1 ... done
Starting inferno_nginx_server_1 ... 
Starting inferno_nginx_server_1 ... done
Attaching to inferno_ruby_server_1, inferno_nginx_server_1
ruby_server_1   | ruby: No such file or directory -- app.rb (LoadError)
nginx_server_1  | 2018/10/02 23:59:13 [emerg] 1#1: host not found in upstream "ruby_server" in /etc/nginx/nginx.conf:72
nginx_server_1  | nginx: [emerg] host not found in upstream "ruby_server" in /etc/nginx/nginx.conf:72
inferno_ruby_server_1 exited with code 1
inferno_nginx_server_1 exited with code 1
```

I don't see where the upstream is define in deployment-files/nginx.conf. Is some pre-configuration expected? Would be good to document.

**Comment by jmandel** [comment]
... and trying to start just the ruby server (e.g., after uncommenting the ports block in the docker-compose file) also fails:

```
$ sudo docker-compose  up ruby_server
Starting inferno_ruby_server_1 ... 
Starting inferno_ruby_server_1 ... done
Attaching to inferno_ruby_server_1
ruby_server_1   | ruby: No such file or directory -- app.rb (LoadError)
inferno_ruby_server_1 exited with code 1
```

**Comment by arscan** [comment]
We caught this a little earlier and fixed it in #70 , could you grab the latest off master and give it another shot?

**Comment by jmandel** [comment]
Lovely - thanks!
